x-pack/test/fleet_api_integration/apis/epm/get_templates_inputs.ts

```diff
@@ -30,6 +33,10 @@ export default function (providerContext: FtrProviderContext) {
     path.dirname(__filename),
     '../fixtures/direct_upload_packages/apache_0.1.4.zip'
   );
+  const istioArchiveZip = path.join(
+    path.dirname(__filename),
+    '../fixtures/test_packages/istio/istio-0.3.3.zip'
```

Comment:
it is a prerelease package? the version do not looks like it to me. Should we use a package from `fixtures/test_packages/prerelease` instead?

Reply:
I think you're right, I'll change the test to use one of those.